Lombiq.Tests.UI/Services/OrchardCoreHosting/OrchardApplicationFactory.cs

```diff
@@ -140,6 +144,18 @@ private void AddFakeStore(IServiceCollection services)
     private static void AddFakeViewCompilerProvider(IServiceCollection services) =>
         services.AddSingleton<IViewCompilerProvider, FakeViewCompilerProvider>();
 
+    // We remove the existing IRecipeHarvester implementations and add a custom implementation that uses the same code
+    // as OC but with a fix in RecipeHarvester.HarvestRecipesAsync to avoid sync over async issue.
```

Comment:
> that uses the same code as OC

Then we shouldn't call it a fake harvester since it actually functions and fake should be reserved to stub/dummy services. How about AsyncRecipeHarvester?